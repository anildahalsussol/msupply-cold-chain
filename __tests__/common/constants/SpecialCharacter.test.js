import { SPECIAL_CHARACTER } from '~constants';

describe('Constants: PERMISSION', () => {
  it('Matches snapshot', () => {
    expect(SPECIAL_CHARACTER).toMatchSnapshot();
  });
});
