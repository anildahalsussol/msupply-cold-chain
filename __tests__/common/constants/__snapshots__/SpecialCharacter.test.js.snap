// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: PERMISSION Matches snapshot 1`] = `
Object {
  "BLACK_CIRCLE": "●",
  "CELSIUS": "C",
  "DASH": "-",
  "DEGREE": "°",
  "DEGREE_CELSIUS": "°C",
}
`;
